docker/README.md

```diff
@@ -13,6 +13,23 @@ of this directory in the past releases for more info.
 For help please try [Fedimint Github Discussions](https://github.com/fedimint/fedimint/discussions)
 or `#mint-ops` channel on [Fedimint's Discord server](https://chat.fedimint.org/).
 
+## System Requirements
+### 3-of-4 Federation with remote Bitcoin backend
+* Memory: 500MB minimum (activate swap in that case), 1GB recommended
```

Comment:
I would recommend 2GB minimum. 800GB used by fedimintd is reported and matches previous data points I remember: https://github.com/fedimint/fedimint/issues/6889#issuecomment-2836947794 . We would need to tune rocksdb more to bring it lower, otherwise there's not a lot of margin room.

Reply:
On an older version I do see peak mem usage of 900MB, but I think a lot of that was fixed in 0.6/0.7, wasn't it?

Reply:
That's still not a lot room for anything else. Maybe it's about the wording. I would not *recommend* anyone to run with just 1GB, that's all.

Reply:
Fixed